public/docs/dart/latest/glossary.jade

```diff
@@ -1,35 +1,56 @@
-include _util-fns
-
-block var-def
-  - var fragPath='../../ts/latest/_fragments/';
-
-!=partial(fragPath + 'glossary-intro')
-!=partial(fragPath + 'glossary-a1')
-!=partial(fragPath + 'glossary-a-2')
-!=partial(fragPath + 'glossary-b-c')
-!=partial(fragPath + 'glossary-d1')
-!=partial(fragPath + 'glossary-d2')
-!=partial(fragPath + 'glossary-e1')
-!=partial(fragPath + 'glossary-e2')
-!=partial(fragPath + 'glossary-f-l')
-!=partial(fragPath + 'glossary-m1')
-//partial(fragPath + 'glossary-m2') not needed in dart
-!=partial(fragPath + 'glossary-n-s-1')
-
-:marked
-  ## snake_case
-
-.l-sub-section
+extends ../../ts/_cache/glossary.jade
+
+block includes
+  include _util-fns
+
+block annotation-defn
+  :marked
+    There are different kinds of annotation in Dart. Unqualified use of the
```

Comment:
Let's reword the first two sentences to remove the "There are" and give a hint as to what other uses of "annotation" might be. (The first sentence stopped me cold, as I tried to recall what the other kinds of annotations were.) Maybe:

When unqualified, _annotation_ refers to a Dart [metadata annotation][metadata] (as opposed to, say, a type annotation).